UniWatch/UnEnrollViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UniWatch.Models;
```

Comment:
This file is also in the root folder of the `UniWatch` solution. Should be moved or deleted.